<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HNS_Message_Edit_Screen</class>
 <widget class="QWidget" name="HNS_Message_Edit_Screen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1140</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="PB_Edit_AddStaticImage">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>340</y>
     <width>162</width>
     <height>107</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#FFFFFF;color:#414141;border-style:outset;border-width:3px;border-color:#1A3760;border-radius:12px</string>
   </property>
   <property name="text">
    <string>ADD SAVED
STATIC IMAGE</string>
   </property>
  </widget>
  <widget class="QToolButton" name="PB_Edit_LeftJustify">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>530</y>
     <width>55</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
    </font>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/Left Line Justify.png</normaloff>:/icons/Left Line Justify.png</iconset>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Edit_Direct_MULTI">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>610</y>
     <width>162</width>
     <height>107</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#FFFFFF;color:#414141;border-style:outset;border-width:3px;border-color:#1A3760;border-radius:12px</string>
   </property>
   <property name="text">
    <string>DIRECT
MULTI
INPUT</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>510</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#414141</string>
   </property>
   <property name="text">
    <string>Page Off Time</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Edit_FlashText">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>580</y>
     <width>162</width>
     <height>107</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#FFFFFF;
	color:#414141;
	border-style:outset;
	border-width:3px;
	border-color:#1A3760;
	border-radius:12px
}

QPushButton:checked{
	background-color:#1A3760;
	color:#FFFFFF;
}</string>
   </property>
   <property name="text">
    <string>FLASHING
TEXT</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="HNS_Message_Preview" name="GV_Edit_Preview" native="true">
   <property name="geometry">
    <rect>
     <x>246</x>
     <y>101</y>
     <width>620</width>
     <height>350</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QToolButton" name="PB_Edit_RightJustify">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>530</y>
     <width>55</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
    </font>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/Right Line Justify.png</normaloff>:/icons/Right Line Justify.png</iconset>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>510</y>
     <width>135</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#414141</string>
   </property>
   <property name="text">
    <string>Page Justification</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QToolButton" name="PB_Edit_CenterLineJustify">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>530</y>
     <width>55</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
    </font>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/Center Line Justify.png</normaloff>:/icons/Center Line Justify.png</iconset>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Edit_AddPage">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>220</y>
     <width>162</width>
     <height>107</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#FFFFFF;color:#414141;border-style:outset;border-width:3px;border-color:#1A3760;border-radius:12px</string>
   </property>
   <property name="text">
    <string>ADD PAGE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Edit_AddMessage">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>220</y>
     <width>162</width>
     <height>107</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#FFFFFF;color:#414141;border-style:outset;border-width:3px;border-color:#1A3760;border-radius:12px</string>
   </property>
   <property name="text">
    <string>ADD SAVED
MESSAGE</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_Edit_PageOffTime">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>530</y>
     <width>132</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>29</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#414141</string>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Edit_Title">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>751</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>33</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#1A3760</string>
   </property>
   <property name="text">
    <string>EDIT CURRENT MESSAGE - PAGE 1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_Edit_PageOnTime">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>530</y>
     <width>132</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>29</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#414141</string>
   </property>
   <property name="text">
    <string>2.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Edit_FontOptions">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>460</y>
     <width>162</width>
     <height>107</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#FFFFFF;color:#414141;border-style:outset;border-width:3px;border-color:#1A3760;border-radius:12px</string>
   </property>
   <property name="text">
    <string>FONT OPTIONS</string>
   </property>
  </widget>
  <widget class="QToolButton" name="PB_Edit_Next">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>250</y>
     <width>44</width>
     <height>37</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton{
background-color:#FFFFFF;
border:none
}</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/Right Arrow.png</normaloff>:/icons/Right Arrow.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>42</width>
     <height>35</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Edit_RemovePage">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>340</y>
     <width>162</width>
     <height>107</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#FFFFFF;color:#414141;border-style:outset;border-width:3px;border-color:#1A3760;border-radius:12px</string>
   </property>
   <property name="text">
    <string>REMOVE
PAGE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Edit_Preview">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>100</y>
     <width>162</width>
     <height>107</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#FFFFFF;
	color:#414141;
	border-style:outset;
	border-width:3px;
	border-color:#1A3760;
	border-radius:12px
}

QPushButton:checked{
	background-color:#1A3760;
	color:#FFFFFF;
}</string>
   </property>
   <property name="text">
    <string>PREVIEW AND
RUN</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="PB_Edit_Prev">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>44</width>
     <height>37</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton{
background-color:#FFFFFF;
border:none
}</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/Left Arrow.png</normaloff>:/icons/Left Arrow.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>42</width>
     <height>35</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="PB_Edit_TopJustify">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>530</y>
     <width>55</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
    </font>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/Top Page Justify.png</normaloff>:/icons/Top Page Justify.png</iconset>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Edit_Cancel">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>640</y>
     <width>181</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#FFFFFF;color:#414141;border-style:outset;border-width:3px;border-color:#1A3760;border-radius:12px</string>
   </property>
   <property name="text">
    <string>CANCEL CURRENT
MESSAGE</string>
   </property>
  </widget>
  <widget class="QToolButton" name="PB_Edit_CenterPageJustify">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>530</y>
     <width>55</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
    </font>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/Center Page Justify.png</normaloff>:/icons/Center Page Justify.png</iconset>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Edit_tostmsg">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>460</y>
     <width>240</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#414141</string>
   </property>
   <property name="text">
    <string>Tap for Keyboard</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Edit_AddAnimatedImage">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>460</y>
     <width>162</width>
     <height>107</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#FFFFFF;color:#414141;border-style:outset;border-width:3px;border-color:#1A3760;border-radius:12px</string>
   </property>
   <property name="text">
    <string>ADD SAVED
ANIMATED
IMAGE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Edit_InsertRadarSpeed">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>162</width>
     <height>107</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#FFFFFF;color:#414141;border-style:outset;border-width:3px;border-color:#1A3760;border-radius:12px</string>
   </property>
   <property name="text">
    <string>INSERT
RADAR SPEED</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Edit_Back">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>640</y>
     <width>163</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#FFFFFF;color:#414141;border-style:outset;border-width:3px;border-color:#1A3760;border-radius:12px</string>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
  </widget>
  <widget class="QToolButton" name="PB_Edit_BottomJustify">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>530</y>
     <width>55</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
    </font>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/Bottom Page Justify.png</normaloff>:/icons/Bottom Page Justify.png</iconset>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>510</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#414141</string>
   </property>
   <property name="text">
    <string>Page On Time</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>510</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#414141</string>
   </property>
   <property name="text">
    <string>Line Justification</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="HNS_Radar_Info_Widget" name="PGE_Radar_Info" native="true">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>610</y>
     <width>280</width>
     <height>157</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HNS_Radar_Info_Widget</class>
   <extends>QWidget</extends>
   <header>hns_radar_info_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>HNS_Message_Preview</class>
   <extends>QWidget</extends>
   <header>hns_message_preview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
